### DISCUSSION TITLE
Beatmap text colour top right should change based on beatmap background

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#26171] [Ideas] [answered]
I know you guys are working on a whole song select redesign so this is a general idea that could probably be applied to both:
On some songs on current lazer that have a white background it's kinda hard to read the beatmaps difficulty when the background is white.
On the redesign this isn't an issue but I don't notice a fade on the beatmap name like there is on current lazer. 

Example of my issue:
![image](https://github.com/ppy/osu/assets/66184829/07e86ee9-e59d-4984-ab66-4b830ed776c1)

> On the redesign this isn't an issue but I don't notice a fade on the beatmap name like there is on current lazer. 

This is just something I noticed, no idea if it's already been thought of or not ^^

Would be nice if text colour changed to black depending on how "bright" the beatmap background is to increase legibility

**Comment by bdach** [comment, marked as answer]
screen will be redesigned soon, hopefully with this in mind
